packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -49,7 +50,7 @@ export const styles = css`
         background-color: transparent;
         height: 100%;
         width: 100%;
-        border: ${borderWidth} solid transparent;
+        border: 1px solid transparent;
```

Comment:
Why is the borderWidth design token no-longer being used?

Reply:
Sorry, I forgot to change that back to the design token, it's fixed now along with a few other places where I was using 1px instead of the token 👍